src/FcmChannel.php

```diff
@@ -28,10 +28,12 @@ public function __construct(Client $client)
      * @param mixed $notifiable
      * @param \Illuminate\Notifications\Notification $notification
      *
+     * @return array|void
```

Comment:
```suggestion
     * @return Response[]
```

(you'll have to import the guzzle response class)

Reply:
Yeah I do something similar for APNS with their response interface. I think that makes sense. The LogNotification event I create that listens to the NotificationSent event will have implementation details specific to the application.